ScoutSuite/providers/aws/metadata.json

```diff
@@ -313,6 +313,13 @@
                 }
             }
         },
+        "docdb": {
+            "resources": {
+                "clusters": {
```

Comment:
I've added this so it now shows in the HTML report, but the path is wrong as it should include `.regions.id.`.